### PR TITLE
Fix/project section

### PR DESCRIPTION
**Pull request by esther921**

### CODE REVIEW

**Review by iamzilfa — APPROVED** [review]

File: components/sections/ProjectSection.jsx [hunk 1]
```diff
@@ -1,113 +1,74 @@
-import React, { useEffect, useState, useRef } from "react";
+import React, { useLayoutEffect, useState } from "react";
 import Image from "next/image";
 import Wrapper from "../wrappers/Wrapper";
 import arrow from "../../public/assets/images/arrow.svg";
+import IMAGES from "../../data/projectImagesData";
+import Link from "next/link";
 
 export default function ProjectSections() {
-  const hover = [
```
Comment (NNesta) [inline components/sections/ProjectSection.jsx:7]: Please use arrow function as it is the recommended one from ES6

File: components/wrappers/Wrapper.jsx [hunk 2]
```diff
@@ -1,8 +1,22 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 
 function Wrapper({ children, styles }) {
+  const [width, setWidth] = useState(null);
```
Comment (NNesta) [inline components/wrappers/Wrapper.jsx:4]: Please use arrow function here

**Review by NNesta — CHANGES_REQUESTED** [review]

**Review by iamzilfa — APPROVED** [review]

**Review by NNesta — APPROVED** [review]
Good job @esther921

**Review by Stevenkwizera06 — APPROVED** [review]
Good work Esther seems you collected all the mistake

**Pull request merged**